### ISSUE TITLE
Ocelot throw 404 when i call any method after update MicroElements.Swashbuckle.FluentValidation from 5.1.0 to 5.2.0

### ISSUE THREAD

**Issue by inyutin-maxim** [micro-elements/MicroElements.Swashbuckle.FluentValidation#93]

**Comment by petriashev** [comment]
Sorry for long response time - I am on vacations now.
I cant see any swagger related request. Swashbuckle adds own middleware to asp net core that responds to the "swagger" endpoint. MicroElements.Swashbuckle.FluentValidation just plugs in schema generation process.

If you remove MicroElements stuff does all work?

**Comment by inyutin-maxim** [comment]
> Sorry for long response time - I am on vacations now.
> I cant see any swagger related request. Swashbuckle adds own middleware to asp net core that responds to the "swagger" endpoint. MicroElements.Swashbuckle.FluentValidation just plugs in schema generation process.
> 
> If you remove MicroElements stuff does all work?

Yes, when I delete your package or downgrade for version 4.3.0, my gateway project work correctly.
Error after installation package version 5.0.0+
